docs/ReleaseHistory.md

```diff
@@ -237,3 +238,55 @@
 
 ## 1.4.15 - 04/16/2024
 - NEW: Implement `IdentifiableSecrets.ComputeDerivedSymmetricKey` to generate identifiable derived keys from arbitrary identifiable secrets.
+
+## 1.4.14 - 04/16/2024
```

Comment:
>14 [](http://example.com/codeflow?start=7&length=2)

OK, so I went back-and-forth, we now have an explicit version and date of release for everything we shipped. I did not retroactively build release notes for release with only logic changes, I only added `RUL` tags (for every shipped rule).

I tend to find the dates helpful and to know everything in play.